<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxHelp">
     <property name="title">
      <string>说明</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QTextEdit" name="txtHelp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidgetSettings">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>外观</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>界面</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="fontComboAppFont">
            <property name="currentFont">
             <font>
              <family>仿宋</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>字号</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinAppFontSize">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>2</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>字体</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>节点列表</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="fontComboListFont">
            <property name="currentFont">
             <font>
              <family>宋体</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>字号</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>字体</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinListFontSize">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>2</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>节点显示格式</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editNodeReprFormat"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>其他</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QCheckBox" name="chkJsonHighlight">
            <property name="text">
             <string>JSON 语法高亮</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Qv2ray</string>
      </attribute>
      <widget class="QCheckBox" name="chkAutoStartQv2ray">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>241</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>自动导入配置后启动Qv2ray</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>551</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Qv2ray 目录</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="editQvFolder"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrwsQvFolder">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabV2ray">
      <attribute name="title">
       <string>v2ray</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxV2rayAdvanced">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>541</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Advanced</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="spacing">
         <number>12</number>
        </property>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>域名匹配算法</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboDomainMatcher">
          <item>
           <property name="text">
            <string>linear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mph</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>负载均衡策略</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBalancerStrategy">
          <item>
           <property name="text">
            <string>random</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>leastPing</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="chkSelectorPrefix">
          <property name="text">
           <string>负载均衡选择器列表精简</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>多端口</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="horizontalSpacing">
        <number>12</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>输出路径</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="verticalSpacing">
           <number>14</number>
          </property>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>为 SwitchyOmega 生成的配置文件</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="editSoResultPath"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnBrwsSoResultPath">
                <property name="text">
                 <string>浏览</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>为 Qv2ray 生成的复杂配置文件</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="editQvComplexConfigResultPath"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnBrwsQvResultPath">
                <property name="text">
                 <string>浏览</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>标签格式</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>路由规则标签</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editInboundTagFmt"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="editRuleTagFmt"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>入站标签</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>出站标签</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editOutboundTagFmt"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBoxMappingReport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>端口映射报告</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>14</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>模板</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="editMappingReportTemplatePath"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnBrwsMappingReportTemplPath">
                <property name="text">
                 <string>浏览</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>导出路径</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="editMappingReportResultPath"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnBrwsMappingReportResultPath">
                <property name="text">
                 <string>浏览</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>入站端口</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <property name="topMargin">
           <number>18</number>
          </property>
          <property name="horizontalSpacing">
           <number>8</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinDefaultPortStart">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>类型</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>默认起始端口</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboPortType">
            <item>
             <property name="text">
              <string>SOCKS5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HTTP</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>负载均衡</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>541</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>标签格式</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>出站标签</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editOutboundTagFmt2"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>541</width>
         <height>101</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>输出路径</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>为 Qv2ray 生成的复杂配置文件</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLineEdit" name="editQvComplexConfigResultPath2"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnBrwsQvResultPath2">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>关于</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="labQv2rayIcon">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/res.qrc">:/3.ico</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labThanksQv2ray">
         <property name="text">
          <string>Thanks To Qv2ray</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAboutQt">
         <property name="text">
          <string>关于 Qt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
